src/main/java/com/training/assignments/Operations.java

```diff
@@ -0,0 +1,102 @@
+package com.training.assignments;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.util.HashMap;
+
+public class Operations {
+    private Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    public String printCompanyReviews(String companyName){
```

Comment:
violating java principles. Having suffix as print but it is returning something.
Lets also create 2 versions of this, one with hashmap and other with switch case